project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.4.7
+sbt.version=1.5.0-RC1
```

Comment:
Not sure if we should update the version already. The changes can be incorporated though.